ci/jenkins/test-vmc.sh

```diff
@@ -0,0 +1,426 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+CLUSTER=""
+DEFAULT_WORKDIR="/var/lib/jenkins"
+DEFAULT_KUBECONFIG_PATH=$DEFAULT_WORKDIR/.kube/config
+WORKDIR=$DEFAULT_WORKDIR
+KUBECONFIG_PATH=$DEFAULT_KUBECONFIG_PATH
+MODE="report"
+RUN_GARBAGE_COLLECTION=false
+RUN_SETUP_ONLY=false
+RUN_CLEANUP_ONLY=false
+TESTCASE=""
+SECRET_EXIST=false
+TEST_FAILURE=false
+CLUSTER_READY=false
+
+_usage="Usage: $0 [--cluster-name <VMCClusterNameToUse>] [--kubeconfig <KubeconfigSavePath>] [--work-home <HomePath>]
+                  [--log-mode <SonobuoyResultLogLevel>] [--testcase <e2e|conformance|whole-conformance|networkpolicy>]
+                  [--garbage-collection] [--setup-only] [--cleanup-only]
+
+Setup a VMC cluster to run K8s e2e community tests (E2e, Conformance, whole Conformance & Network Policy).
+
+        --cluster-name           The cluster name to be used for the generated VMC cluster.
+        --kubeconfig             Path to save kubeconfig of generated VMC cluster.
+        --work-home              Home path for Go, vSphere information and antrea_logs during cluster setup. Default is $WORKDIR.
```

Comment:
should `--work-hom` be changed to `--workdir`?

Reply:
Thanks for reminding me. Yes, it should be changed.